### PR TITLE
ST6RI-296 Publish jupyter-sysml-kernel to conda-forge

### PR DESCRIPTION
**Pull request by ivan-gomes**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ivan-gomes** [comment]
@staged-recipes I am ready for your review of this new recipe. Thanks in advance!

@conda-forge-admin, please ping team

File: recipes/jupyter-sysml-kernel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} ./install.py --api-base-path=http://sysml2.intercax.com:9000 --sys-prefix
+
+requirements:
+  build:
+    - jupyter_client
+    - python >=3
```
Comment (mariusvniekerk) [inline recipes/jupyter-sysml-kernel/meta.yaml:21]:
```suggestion
```

File: recipes/jupyter-sysml-kernel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  host:
+    - jupyter_client
+    - python >=3
```
Comment (mariusvniekerk) [inline recipes/jupyter-sysml-kernel/meta.yaml:24]:
```suggestion
    - python >=3.6
```

File: recipes/jupyter-sysml-kernel/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  run:
+    - graphviz
+    - jupyter_client
+    - openjdk >=8
+    - python >=3
```
Comment (mariusvniekerk) [inline recipes/jupyter-sysml-kernel/meta.yaml:29]:
```suggestion
    - python >=3.6
```

**Comment by ivan-gomes** [comment]
Thanks for the review @mariusvniekerk. I have merged in all your suggestions.

@conda-forge-admin, please ping team

**Comment by ivan-gomes** [comment]
I hope pinging every couple weeks seeking a review is acceptable. Really hoping to move forward with this. Thanks in advance!

@conda-forge-admin, please ping team

**Comment by ivan-gomes** [comment]
Apologies for the re-tag, but would _really_ like to get this merged in asap. Thanks!

@mariusvniekerk 
@conda-forge-admin, please ping team

**Comment by ivan-gomes** [comment]
@conda-forge-admin, please ping team

**Pull request merged**